src/services/docker/docker-image.go

```diff
@@ -0,0 +1,27 @@
+package docker
+
+import (
+    "fmt"
+	"github.com/nearform/k8s-kurated-addons-cli/src/utils/defaults"
+)
+
+type DockerImage struct {
+    Registry    string
+    Name        string
+    Directory   string
+    Tag         string
+}
+
+func (di DockerImage) RemoteTag() string {
```

Comment:
can we use dockerImage instead of the parameter di to keep the parameter name consistent between here and in src/services/docker/docker.go